torch/testing/_internal/common_methods_invocations.py

```diff
@@ -209,6 +209,8 @@ def to_numpy(t):
             if isinstance(t, torch.Tensor):
                 if t.dtype is torch.bfloat16:
                     return t.detach().cpu().to(torch.float32).numpy()
+                if t.dtype is torch.chalf:
```

Comment:
It might be nice to say something about the dtype of the returned ndarray in `.numpy()`'s doc since we upcast some dtypes like bfloat16 and now chalf when there's no equivalent dtype supported in numpy https://pytorch.org/docs/master/generated/torch.Tensor.numpy.html#torch.Tensor.numpy